cfme/fixtures/nuage.py

```diff
@@ -58,3 +58,26 @@ def with_nuage_sandbox(networks_provider):
     enterprise = sandbox['enterprise']
     nuage.delete_enterprise(enterprise)
     logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
+
+
+@pytest.fixture(scope='module')
+def with_nuage_sandbox_modscope(appliance, setup_provider_modscope, provider):
+    nuage = provider.mgmt
+    sandbox = create_basic_sandbox(nuage)
+    logger.info('Performing a full refresh, this might take a while.')
```

Comment:
Please include more context here in the log message - your message on L83 is perfect, including some information about the sandbox.